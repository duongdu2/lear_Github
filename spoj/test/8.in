46
17
1335 3415 409 1643 987 123 4419 6207 2096 8386 7933 8809 3037 2235 4327 4652 1535 
29
81 8526 3737 3153 3365 3461 6174 5141 416 957 7220 3017 8191 2775 1256 2938 1790 6438 5095 3883 3186 4005 4563 601 7043 6657 606 6902 2293 
38
5008 1364 8343 5322 9380 6559 2729 7451 9602 5050 149 2685 6658 7282 7896 5096 5250 5522 5463 4096 7556 5041 1707 6544 1923 7108 6294 9161 8691 6476 1607 2848 2107 9874 5888 144 4807 3210 
46
2762 9568 2268 4908 5625 9798 7422 9965 8941 9782 3630 3440 7869 5928 2437 5127 3310 5535 2317 8901 1344 9740 5969 8937 8697 6234 7502 1291 8117 3530 9603 7710 1499 3902 5965 4587 8779 9976 8672 8017 7974 8220 8516 1936 5476 6384 
23
8991 2576 5379 5193 1580 8609 7020 628 8569 4468 929 8551 1109 6250 5056 2205 189 3380 6499 8063 9112 6758 247 
16
4267 9473 8644 5291 8443 3138 4184 1099 1701 6617 6844 4930 6671 9958 6238 1405 
42
9145 3228 5429 6909 5459 4661 7484 9530 8544 4095 3615 3081 7676 4798 7624 389 8559 9057 1310 8580 8532 8887 1269 2596 5006 5637 8722 6132 2293 173 42 5441 6898 2 1485 2443 5869 5694 9209 2703 6511 7769 
37
9545 3664 126 4127 4993 6544 2338 3889 4526 6568 3401 1205 9915 6966 6511 6776 7902 2791 3380 6774 7068 881 6924 2500 1314 7029 1262 106 6176 4813 6908 109 4632 710 6912 9021 7693 
17
5030 3116 8121 487 5315 6250 9308 6518 2322 3191 4331 2535 9757 8283 3554 8569 8557 
32
4901 23 978 1434 1550 4256 8100 1097 3557 7170 9956 4369 9504 9374 6536 6129 4311 1905 6848 4525 6834 1987 6334 5061 602 7370 3335 3177 618 1762 9839 3023 
17
1809 2128 3093 1714 751 2703 8944 4154 4198 8908 7679 999 5664 301 9480 5324 6049 
25
3428 3736 530 3794 3281 7073 2071 6972 6606 6222 8532 1295 2360 8175 520 1194 962 2330 6853 1075 1709 4980 3453 1406 1623 
48
6369 5050 9480 6277 546 6932 3524 5370 3586 2406 898 5297 3662 8666 2792 675 6296 8425 6979 4904 9560 8663 2041 349 8704 614 3860 2564 5408 7815 8619 1437 873 1388 7107 5477 516 8069 6234 417 6070 83 6431 4379 2855 9695 8237 9441 
42
7407 6379 394 861 7988 9016 9974 9289 3643 8745 404 9871 8234 8984 6948 7409 8263 9921 4389 932 689 5114 7339 1167 9043 5650 4789 2449 1127 2910 9970 1371 7931 9200 7921 6394 2370 2798 1111 5164 1374 9472 
3
8857 997 4397 
16
1354 9445 9884 1884 7152 3518 6446 6509 9179 5212 7007 539 4406 6022 6599 6284 
48
3511 205 5091 7425 6116 9529 6205 4399 2885 5525 7634 5413 933 8522 4439 2555 4341 3352 3166 9918 5522 8748 8036 2862 8459 2936 477 4683 6388 5921 241 9417 4296 5681 9940 2870 8863 1091 5737 5877 8007 4477 5992 5675 6884 8411 1843 1533 
23
7831 6311 4634 228 5443 6688 2075 8323 8176 7926 7710 3672 992 1295 1227 7067 133 809 4200 7958 6770 8179 4938 
33
1088 5326 8312 4180 1917 3429 1676 9312 8509 2797 1316 8119 1694 127 8755 1485 7685 2331 5329 696 8511 1465 579 7925 2892 5047 5213 3011 6845 8518 6180 148 5234 
41
7235 5540 6616 4906 9009 4700 7562 2790 5909 9587 702 981 788 3660 4823 9738 9468 3480 1325 3415 4520 8811 4244 778 6349 6743 8874 3096 4171 2304 4314 1901 1103 4107 8932 7914 9034 8180 397 8623 4060 
28
943 3113 3121 8230 9270 4420 8519 8507 2438 655 3194 2682 2519 6046 9829 2634 3825 4174 3501 6177 8459 3956 1286 3161 5150 5215 8774 1000 
39
591 7464 9275 7201 9702 8062 709 9818 8703 6808 8146 1913 2547 6556 4861 2451 4130 8406 2679 7911 9604 6178 8410 5017 6009 6206 6590 9046 4413 9422 6281 2994 5665 2525 9535 7904 9134 2350 8964 
22
2387 2681 9965 5758 5436 3215 7316 8166 9415 8384 4148 7456 2868 964 7981 7984 2689 9868 220 1906 1461 4354 
43
4964 9293 6589 5371 5252 9127 7025 9761 250 3012 7894 6767 1851 5488 7093 5877 236 3569 5858 5237 9293 7119 9407 6334 625 2586 1483 795 6714 9780 2227 573 8747 3219 7248 5069 9488 4858 5881 2311 7376 2162 3232 
30
5064 4736 1300 7369 5026 7671 4026 2053 4976 404 1761 977 9297 5927 2586 4072 1080 1718 4108 639 4384 4031 649 3414 7735 1753 2860 7640 1696 5939 
26
4570 5025 7614 7125 4672 5660 9080 9098 2549 4553 7903 666 3058 6884 4123 8871 5757 141 8313 7701 8284 7048 8205 8647 1017 8227 
35
1086 1664 9008 5362 5286 660 8267 4589 9111 1332 6986 6745 4524 9072 6712 6752 5449 5623 7097 4240 9718 9257 9536 4847 5446 668 2161 8152 5070 5790 2244 8211 4621 7584 1954 
38
25 6896 4841 8714 155 8101 8449 1877 628 3035 8057 3957 1981 888 1512 6364 7098 366 7928 5745 3272 6650 3334 4536 6018 6000 2720 2954 6143 9561 3932 6645 6516 7637 3014 178 3742 6982 
5
3949 7937 5317 9093 8971 
31
3874 3433 1836 6790 6878 2963 1237 4424 3403 4656 2696 6648 5907 6889 4119 2088 9372 3 3115 3307 2004 6902 7047 6636 9802 3398 9519 6298 7401 5170 2266 
13
6628 8311 3825 5400 6882 5194 164 6321 8013 9282 521 7702 278 
16
2594 9447 1449 4319 5991 5148 1525 1198 5256 9296 688 3878 2354 1175 9321 8555 
13
9991 5678 6545 8071 5967 5828 7460 8294 9506 3146 6878 4519 5530 
46
4389 7579 3379 1833 460 231 1622 2343 6291 4424 6249 6836 6785 3337 277 4506 2444 1391 3710 4486 4683 5817 8214 7877 4190 7654 7736 7611 7695 7438 6773 2932 586 2583 1032 50 353 476 8470 9471 1008 5532 9208 206 3231 3126 
40
2273 4924 535 6853 4702 8077 190 7212 765 7221 391 6768 6373 1736 3895 1182 4526 9462 9611 5082 3113 2966 2908 1970 6023 9627 1558 1528 2594 5896 489 1040 8595 2882 3506 8406 1739 28 4485 6192 
34
5541 8331 8130 9617 7766 9369 276 3602 158 8350 6042 9863 1690 7001 6721 3460 1839 5951 9821 6310 4216 5814 8944 7597 9434 6692 8903 6158 9564 6696 1337 9900 5480 6893 
35
938 8790 6562 667 8115 9317 2406 6802 4821 7676 886 9945 7592 4305 3417 875 9251 4889 2134 3958 9421 6533 3586 9242 4261 8122 418 8496 3423 8396 9536 176 5440 9148 9212 
32
4601 9658 7847 9335 8711 7486 481 5218 1465 2600 3234 9890 5481 467 8151 5318 6777 3098 4179 3395 744 4445 6205 7501 4293 1087 4478 5148 2290 9640 9462 608 
19
960 2173 3960 5488 2048 907 3730 6136 7444 731 5613 4997 2817 9795 66 821 184 6201 2237 
36
2173 700 138 7097 9068 8522 1909 3766 1535 8294 9651 9873 271 8132 303 2052 7148 9633 9712 2647 746 2832 2913 5312 2785 9605 5646 801 8005 3517 2446 445 6032 5578 6372 913 
8
4534 1566 9276 8807 2773 2869 3696 5350 
17
2634 5288 5793 4028 2160 3113 2839 9962 8985 1049 342 3310 4701 4120 760 8454 7226 
32
7973 5700 735 9378 6287 8853 4237 8193 5724 2455 9245 527 5915 5136 8491 4826 5691 4904 7587 615 7346 8692 3813 5279 3899 9032 7815 7186 7257 1910 8016 8667 
22
332 5820 3965 8881 46 4339 6239 2166 7173 4428 374 692 4762 4593 3360 2552 5246 8586 5021 8416 1726 8153 
3
9004 8003 1162 
5
5192 7435 3679 6485 4330 
