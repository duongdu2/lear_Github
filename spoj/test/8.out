25128
23937
28856
29739
26712
28075
27884
28481
27634
29299
27332
23780
28735
29865
14251
28508
29387
24678
26585
28793
27873
29124
29248
29029
24703
27049
28511
26724
26001
28693
25606
28064
27791
27149
28700
29584
28902
29188
23440
29236
23433
27401
27655
25883
18169
19112
